misc/python/materialize/mzcompose/__init__.py

```diff
@@ -93,7 +93,6 @@
     "persist_roundtrip_spine": "true",
     "persist_sink_minimum_batch_updates": "128",
     "persist_stats_audit_percent": "100",
-    "persist_txn_tables": "lazy",
```

Comment:
This will also apply to older versions now. So while we used to compare lazy vs lazy we now compare defaults: eager in old versions against lazy in new. I'd propose to keep this line in, then the benchmarks should be green.